README.md

```diff
@@ -214,6 +214,75 @@ Custom translations of properties can be added by extending the
 (def datasource-options {:jdbc-url "jdbc:sqlite:db/database.db"})
 ```
 
+### ClickHouse example
+`hikari-cp` can be used with [official ClickHouse driver](https://github.com/ClickHouse/clickhouse-java) in following two ways:
```

Comment:
What are the pros and cons of these two ways? Maybe one preferred way of doing things would be better?

Reply:
`com.clickhouse.jdbc.ClickHouseDataSource` allows additional driver level customisations to be defined via a 2 arity constructor that takes additional `Properties` object.

Using `:jdbc-url` with `:driver-class-name` set as `com.clickhouse.jdbc.ClickHouseDriver`, creates `com.zaxxer.hikari.util.DriverDataSource` object. 

The only difference between the two is the additional customisations that can be specified and are understood by the `ClickHouseDataSource`.

Datasource customizations would be needed when running large queries or performing huge bulk inserts in ClickHouse